_data/other.yml

```
@@ -373,6 +373,7 @@ websites:
       - phone
       - totp
     doc: https://help.kickstarter.com/hc/en-us/articles/115005127094
+    exception: "Currently requires a verified SMS Number as fallback for Software 2FA."
```

```suggestion
    exception: "Software 2FA requires setting up SMS 2FA as a fallback."
```

I think that this reads slightly better?

It sounds more similar to existing exception text as well: E.g. for Dropbox "Hardware 2FA requires enabling Software or SMS 2FA as a back-up."

---

Thank you. 
It is much clearer that way.